connector/connect/server/src/main/scala/org/apache/spark/sql/connect/artifact/SparkConnectArtifactManager.scala

```diff
@@ -82,6 +83,40 @@ class SparkConnectArtifactManager(sessionHolder: SessionHolder) extends Logging
    */
   def getSparkConnectPythonIncludes: Seq[String] = pythonIncludeList.asScala.toSeq
 
+  private def areFilesEqual(path1: Path, path2: Path): Boolean = {
```

Comment:
Can't you re-use some functions from libs like Apache Common: `IOUtils::contentEquals` or `FileUtils.contentEquals`. Or there is some reason to don't do that. Please, clarify that.

Reply:
That sounds good, let me see if we can use it

Reply:
Added this, thanks for the feedback!